### PR TITLE
BLUEBUTTON-1792: Add AB2D certificates for CCS DPR

### PR DESCRIPTION
**Pull request by dkrylovsb**
This PR adds the necessary certificates for the AB2D lower environment to connect to the BFD DPR environment. Self signed certs are attached and added to the env_specific.yml file in the playbooks-ccs directory.

### CODE REVIEW

**Comment by njdister** [comment]
The following peering work needs to be done by CCS before this work can be tested: https://jira.cms.gov/browse/CMSAWSOPS-53861

**Comment by karlmdavis** [comment]
@jzulim , @njdister , or @cthulhuplus: Could you take a look at one of the older similar PRs I've reviewed and run this one through the same checks I used on those?

For example, see my resolved comments on this one: https://github.com/CMSgov/beneficiary-fhir-data/pull/16

Those five checks should be treated as firm requirements.

File: ops/ansible/playbooks-ccs/files/client_data_server_ab2d_dev_certificate.pem [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+WtFqZGeNTrUxEFXcUEtV9AWeZnU512Gmw9cAmEfQ8LDxc1bwK69Ar4jyagpwfnfs
+uTF/Zr6uGexYZ0IbO+dFo/4CNO6krPA4vTcmX9BP+mIHgxEL0py4tAbf3ZDOunVM
+/RPl01s4ezes/5yomXHrI1LegrIHUKE5JJRh1KRTo/vPQF2Jy50856Qwa2gZh5Pn
+6fAbfJh6tKtuKe7pEuS3Cl6Uhhe4pc7rXnvjia3n8AOb4Z39d9Vss4vDKH2zI0uY
+mfSI4M3pbEJqMisLaHWWUJYTtXTV94qQzamNjSXj3AM2PoDuTgSXDkP8oirvJcUb
+VQAiks62zRB1HW0e+X+w/IOyW9CLSuli/faQPIar8CC5ANspfxYuWaaXwHhm2D4X
+5iDRX2QxPkvY1T/bJh3JLufMYA+QPsXZypdW8x1LBayz5cBHdv8OwDjhhZtw+3cd
+loenVP0hmbq9x/RUwNv1mxPwen0WM0zPrcEl6Rg7rrkUqxHEHoUun0YwUBSAZlO8
+EB/xJA==
+-----END CERTIFICATE-----
```
Comment (njdister) [inline ops/ansible/playbooks-ccs/files/client_data_server_ab2d_dev_certificate.pem]:
```
openssl x509 -text -in ops/ansible/playbooks-ccs/files/client_data_server_ab2d_dev_certificate.pem
Certificate:
    Data:
        Version: 1 (0x0)
        Serial Number: 16678885620593733168 (0xe7774e1a78dc2e30)
    Signature Algorithm: sha256WithRSAEncryption
        Issuer: CN=ab2d-dev-client
        Validity
            Not Before: Jan 23 14:39:04 2020 GMT
            Not After : Jan 21 14:39:04 2025 GMT
        Subject: CN=ab2d-dev-client
        Subject Public Key Info:
            Public Key Algorithm: rsaEncryption
                Public-Key: (4096 bit)
                Modulus:
                    00:aa:50:dc:a8:28:b1:49:69:98:42:be:61:79:59:
                    26:d5:d2:f2:b0:db:4f:8d:fc:e4:0f:56:bd:0c:3d:
                    0f:d5:ba:c4:fd:9e:6a:49:20:d6:f3:37:f2:c8:80:
                    25:d2:d5:c0:69:e9:e5:17:f6:b1:31:1a:23:06:17:
                    2c:9d:73:c5:37:d8:65:aa:80:15:38:15:ed:c5:2f:
                    d1:8b:b1:ae:28:ba:e6:e5:35:c4:49:53:e4:36:c6:
                    7e:75:38:e8:16:5f:de:99:9f:87:50:31:ee:9a:2e:
                    8d:e9:2b:1e:fd:b3:39:3a:2e:28:ca:d9:49:04:7c:
                    fe:79:e2:04:32:44:0c:76:fd:75:0a:fb:1a:23:38:
                    61:8d:94:cc:bf:d5:6d:12:8a:d0:2d:64:e3:d5:f5:
                    47:10:7f:ce:4c:4f:5a:1f:d6:d5:c8:1f:78:4d:43:
                    24:4c:d1:b7:a7:3f:75:18:26:a7:3f:25:4c:1d:d6:
                    8c:22:06:67:54:2f:75:c1:07:30:42:99:07:8e:aa:
                    de:79:5f:57:da:e8:ed:44:f2:2c:6e:25:6a:21:98:
                    0e:71:c7:42:ef:d4:02:ae:c2:91:c4:bf:f8:1d:f1:
                    4c:3a:d0:5c:49:39:73:3d:70:f3:08:1e:49:e6:23:
                    46:53:21:db:b3:67:04:98:7b:42:36:8d:7f:70:f9:
                    ff:60:89:c1:f1:00:4d:4f:f4:b2:e5:16:17:d5:4d:
                    bf:60:20:dc:b0:98:1d:81:d6:64:3b:37:93:4f:23:
                    b1:25:d3:55:f0:2b:6b:89:fb:60:5c:54:a0:6a:9c:
                    3e:9f:f4:02:f2:06:8d:b1:b6:13:fe:24:d6:cc:c3:
                    c6:3f:d6:0d:15:f9:41:67:02:db:bd:79:2f:75:f2:
                    ab:18:74:94:41:4b:30:5f:f0:ff:cd:11:99:ec:82:
                    a6:91:47:7c:ef:1d:eb:0a:24:43:79:1c:3b:50:6e:
                    f2:c0:7b:d9:c2:9c:63:b1:34:a2:a5:90:19:9c:9c:
                    9b:a8:78:9c:47:e9:57:9c:27:2d:2c:81:31:88:c2:
                    c0:b3:f5:16:23:22:b5:76:8a:34:48:dd:24:a9:80:
                    f4:cc:a5:fe:4b:83:58:ac:50:a6:40:c4:f7:1e:45:
                    19:ee:4d:39:31:9a:64:7f:ed:02:ab:ea:d6:54:9e:
                    a8:5e:0a:6f:f6:fe:dd:8d:9e:70:b9:3a:c9:fa:4f:
                    2e:24:0b:1a:9f:61:40:d9:75:1b:0a:88:41:3c:3a:
                    20:c1:24:72:60:73:c4:a3:c7:c4:3c:c3:82:99:b2:
                    dd:39:31:46:51:cd:78:33:2d:df:a8:af:39:f5:37:
                    c3:25:fa:4d:b1:b4:d0:6a:87:28:cc:14:00:8d:68:
                    92:c6:27
                Exponent: 65537 (0x10001)
    Signature Algorithm: sha256WithRSAEncryption
```
There's no private key material here, right? ✅ 
Is the cert different from any other certs from the same project (just watching out for oopses)? ✅ 
Is its Subject unique? ✅ 
Does it have a reasonably useful/informative Subject? ✅ 
Does it have a reasonable key size (>= 2048)? ✅

File: ops/ansible/playbooks-ccs/files/client_data_server_ab2d_imp_certificate.pem [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+t0fHDeiGu7pAI6+H2nJuUZmX20EXYns6zUR0Z4hMWKB8HTtqYE3hu42zOFpajtG9
+aAnspwnb3mv66QN1gFwPDpF7yhzA8yZKTud+gWCKvHUDJcmSjgQ16cRe/8IP6h0L
+xQn620jeSJsstkIMKtF/Cyh2yZagou3SeKMty2VklRXqRuNh7ufIVdyc+HbALYGb
+MbwVv2DsdykMD3zkOaWDrPQ2sLmFknRSAuLlZS1iuBoAtMCTTxFEkmRK61gGcJJi
+8hmDTIbxtA1jDwpic2lB2yRjYnddesYPuOsEiLhKoErBb4aoj7ebcAe6yNajNM/3
+qBM/TV4zxvlP4VW52XhQTEOY8sR/KZe4KyCfsQLSkRZaRruCv3egDpXjCMbodUrz
+YAKHhN+39X3tlBIyfQIn6t02zOksrAzIZZ23iOTIqOLs0vzSrxsRD0pbmF+KjFcn
+xRsAXUHSKEyIBJt4aSIEBaR01X3pxveAq0PyM5uR156thHJmIsVZFdRvHbotbuQL
+m2Xpzw==
+-----END CERTIFICATE-----
```
Comment (njdister) [inline ops/ansible/playbooks-ccs/files/client_data_server_ab2d_imp_certificate.pem]:
```
openssl x509 -text -in ops/ansible/playbooks-ccs/files/client_data_server_ab2d_imp_certificate.pem
Certificate:
    Data:
        Version: 1 (0x0)
        Serial Number: 15376305221454973735 (0xd5639c21bacecb27)
    Signature Algorithm: sha256WithRSAEncryption
        Issuer: CN=ab2d-imp-client
        Validity
            Not Before: Jan 23 14:48:38 2020 GMT
            Not After : Jan 21 14:48:38 2025 GMT
        Subject: CN=ab2d-imp-client
        Subject Public Key Info:
            Public Key Algorithm: rsaEncryption
                Public-Key: (4096 bit)
                Modulus:
                    00:9c:b1:70:19:23:86:1f:48:ad:ec:a0:63:66:ef:
                    d7:a0:eb:00:fb:db:7c:63:d3:ed:b1:89:20:2a:b0:
                    52:cd:de:54:28:96:e2:be:d0:25:4d:13:8d:25:7f:
                    ac:4a:19:21:cb:9c:49:b3:ca:f2:38:5f:64:81:df:
                    e0:26:f0:20:0a:49:7c:e4:8f:84:50:a8:3d:96:6b:
                    1c:f2:4d:c7:1c:e6:25:e1:86:82:09:cd:5b:08:6a:
                    ce:4e:a0:fe:2e:71:ee:6c:db:f6:3f:00:7d:a5:d3:
                    8c:82:ea:9c:ed:13:22:ff:0e:95:1e:c6:19:31:35:
                    c2:23:e7:ab:05:a8:2a:e0:c9:08:25:5b:5d:7a:6e:
                    56:7b:5b:91:12:87:73:30:18:5f:68:87:9a:71:8d:
                    4e:48:75:de:0a:25:2b:3d:7a:40:57:cc:02:73:2a:
                    ba:3f:4b:8e:f2:58:31:e7:df:da:15:bd:33:58:92:
                    3a:92:73:95:42:5b:5c:9d:7d:1f:ea:27:10:cc:c5:
                    ef:06:98:83:6b:4f:f1:08:46:29:7d:0d:e5:04:93:
                    66:bb:dd:75:29:66:5e:1a:64:76:9f:e0:c7:de:72:
                    74:11:17:3b:4e:b4:92:d6:f8:8f:18:1e:46:73:e1:
                    a1:ee:92:69:1f:10:14:c0:2c:8a:cd:12:7a:86:08:
                    29:1d:ff:79:f0:47:95:07:a5:12:af:43:f5:65:a9:
                    19:97:ad:b7:43:ab:16:e1:51:16:0d:90:ea:ae:d2:
                    62:6f:6d:16:1c:ef:55:a9:cb:e2:e0:04:a6:57:4a:
                    fd:c2:a1:8e:27:a4:c3:58:3f:af:9e:87:86:28:1e:
                    92:91:63:3c:28:4f:ad:d9:fa:f7:cc:ba:54:49:02:
                    b0:7d:29:cb:93:06:ba:bf:a1:7e:c7:4c:a7:b8:f6:
                    e9:09:fb:0d:10:6b:af:a0:44:4b:4f:88:61:fc:81:
                    17:02:76:8c:d3:b8:33:85:14:10:67:c4:f2:22:e7:
                    35:3c:42:ed:e7:77:92:c5:96:9a:6b:e4:06:36:af:
                    88:bd:61:6e:9f:ae:67:4a:88:a9:5a:b8:c7:42:51:
                    50:62:89:f4:c7:2f:2e:05:cf:dc:8c:13:ab:26:91:
                    69:b2:e5:65:71:ac:8c:d5:71:2c:24:43:67:82:61:
                    0b:fd:7f:db:9b:65:2e:ce:13:91:99:c8:4b:0f:4a:
                    2b:85:55:5d:e8:cd:09:81:e2:c7:ef:3b:84:5a:81:
                    df:fe:ed:51:14:b5:9f:9c:ff:a5:bf:7d:7f:ea:3d:
                    eb:d2:84:69:cd:4a:e6:e2:30:2a:f0:02:7c:33:67:
                    e3:c9:a1:7f:32:e3:e9:26:1b:5d:e0:d7:6e:d0:1b:
                    a7:0b:f5
                Exponent: 65537 (0x10001)
    Signature Algorithm: sha256WithRSAEncryption
```
There's no private key material here, right? ✅
Is the cert different from any other certs from the same project (just watching out for oopses)? ✅
Is its Subject unique? ✅
Does it have a reasonably useful/informative Subject? ✅
Does it have a reasonable key size (>= 2048)? ✅

File: ops/ansible/playbooks-ccs/files/client_data_server_ab2d_sbx_certificate.pem [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+55H79nJd5VU9jiEazF4T+lvCqOEn+b4NoaI+3t6dfnfjsF0uvdw4dKQ3KR0eNNNE
+zUdC/tOi1BHlWBXZg1Mz246p+tGilHI91fQ334MLLY56Ed5GiJOl3/T++/GrFEpp
+Vxx1ZLxrbceBZB7LC6sJVRfZSf3BM3xo06GrmL1nGJQNR+0XEMDIqGYbt6iAuPqd
+WSRiP6txnlrLLhZK/SytOnAwBYE0kKupuFpYH0QXuBn1SR4VqSi78pGb4otfSydu
+KBZW8d5YRPzrBvUUzHzwr26x7V+8mDMYzGVzPrbzm1pztj3wjGu5SyPfPsD7zNHV
+NhpgXk/Q80Pevcu9nR1l1L44VlQQCTCi9VttkQHuPwOUWEeZ2Hn3y259+/EGvlzr
+ulaoFugpy9q4EaNqZzQmh9sOYI9Vk6SKHwQvlOhQt9gRBNjLA6j+7vkWLUbVCFZx
+MdoTmtkGSzp1/1TcIhcGxwUqyk/jFKYvsoTsIWaUmdGb50QyHBIgfPj2AU1xaARb
+67UkIw==
+-----END CERTIFICATE-----
```
Comment (njdister) [inline ops/ansible/playbooks-ccs/files/client_data_server_ab2d_sbx_certificate.pem]:
```
openssl x509 -text -in ops/ansible/playbooks-ccs/files/client_data_server_ab2d_sbx_certificate.pem
Certificate:
    Data:
        Version: 1 (0x0)
        Serial Number: 17737621763724056112 (0xf628b10423029630)
    Signature Algorithm: sha256WithRSAEncryption
        Issuer: CN=ab2d-sbx-client
        Validity
            Not Before: Jan 23 14:47:23 2020 GMT
            Not After : Jan 21 14:47:23 2025 GMT
        Subject: CN=ab2d-sbx-client
        Subject Public Key Info:
            Public Key Algorithm: rsaEncryption
                Public-Key: (4096 bit)
                Modulus:
                    00:da:63:c5:09:c3:9f:22:82:f5:07:54:e1:43:ee:
                    5f:1c:bb:ff:f2:a9:a3:7f:8b:67:b1:21:74:ee:e5:
                    48:a9:01:af:bb:32:60:b9:62:4c:96:18:c8:13:84:
                    b7:8e:d7:5d:ee:ac:e5:5b:ae:cb:b0:47:7e:f9:26:
                    a9:51:22:fa:a8:e9:a4:62:4b:a5:00:a0:98:32:39:
                    d2:bb:51:9d:1d:d6:84:b3:c2:1b:09:c7:40:df:62:
                    aa:9d:a3:17:06:78:ee:da:47:ef:99:e1:1d:ab:8a:
                    aa:38:4b:79:c8:39:a6:88:83:e5:f3:c7:f4:80:94:
                    20:85:18:07:47:41:24:98:b6:11:5e:dc:45:86:85:
                    59:e6:07:20:3d:14:5a:6e:0f:58:46:33:d4:4f:21:
                    df:5d:25:7e:33:f9:44:65:51:0b:b0:36:af:10:31:
                    16:46:fc:52:bd:ad:cf:32:52:ec:03:1c:fe:68:c4:
                    18:6b:8e:b1:6d:e6:6e:b1:ed:10:4d:a0:cc:fd:ce:
                    45:db:e8:47:a1:1b:e6:f7:f7:87:88:75:bf:ac:c3:
                    4e:dd:ae:35:c6:86:11:56:83:0f:ab:94:d5:40:b7:
                    33:cb:20:a7:fd:cc:18:d6:57:d2:5c:82:54:c2:f6:
                    8f:09:29:79:b7:36:9e:aa:ad:c9:1d:3e:3f:a5:2e:
                    25:9b:c3:d3:04:1f:23:b9:20:e5:94:7e:46:39:22:
                    52:9e:6d:86:91:0a:75:08:10:c1:27:64:b0:dd:d7:
                    0c:5a:da:a2:d0:e3:ae:fa:d5:da:af:6b:a5:17:3a:
                    4a:c0:ad:51:b9:db:c9:c9:47:bb:07:cd:5f:b1:4c:
                    d5:12:e4:37:0f:1e:1f:3f:83:77:f6:82:df:31:4c:
                    dc:7c:0e:eb:47:18:5c:84:ca:a0:eb:fc:88:e9:85:
                    df:a0:37:96:fc:29:9c:a9:54:7e:87:0b:9d:e5:7e:
                    ca:7b:68:a0:36:60:ba:98:de:35:69:d3:1c:d4:c6:
                    9c:87:f5:d8:a9:4e:d7:7d:aa:1e:74:28:bd:b5:08:
                    0e:15:ec:32:8e:5f:6a:dc:38:e3:1f:30:35:59:cd:
                    0e:f7:e4:77:fc:c3:dd:5b:08:3c:0e:7f:6b:64:3c:
                    4b:bc:72:88:a7:5e:42:19:9e:20:0a:84:37:fc:b3:
                    dc:b8:b8:0f:a7:a6:05:f0:09:fb:9d:1d:fd:85:f8:
                    99:cd:93:cd:a2:24:49:8d:14:c4:42:59:ea:83:f7:
                    63:fc:32:b6:91:b8:b3:ba:57:41:73:98:00:91:b3:
                    e8:70:79:ba:3f:a5:3d:cf:51:5a:b5:10:8a:10:a1:
                    c3:f3:cf:4e:77:c1:5d:42:ea:46:68:4e:43:4a:15:
                    b7:f4:bb
                Exponent: 65537 (0x10001)
    Signature Algorithm: sha256WithRSAEncryption
```
There's no private key material here, right? ✅
Is the cert different from any other certs from the same project (just watching out for oopses)? ✅
Is its Subject unique? ✅
Does it have a reasonably useful/informative Subject? ✅
Does it have a reasonable key size (>= 2048)? ✅

**Comment by karlmdavis** [comment]
@njdister Just for the record (and folks' future copy-pasta), could you add a comment with details on those checks, like I did?

**Comment by karlmdavis** [comment]
> @njdister Just for the record (and folks' future copy-pasta), could you add a comment with details on those checks, like I did?

Disregard. I'm dumb.

**Pull request merged**